varnish-operator/templates/grafana-dashboard.yaml

```diff
@@ -644,7 +644,7 @@ data:
         },
         "yaxes": [
         {
-          "format": "s",
+          "format": "percent",
```

Comment:
the metrics itself are returned in seconds (`process_cpu_seconds_total`). You sure grafana will convert automatically seconds to percents correctly?

Reply:
Yea I'm not sure about this but time does not seem like the correct unit. Do you know of another metric we can use that returns CPU usage as a percentage?

Reply:
it's correct, just a different way - it tells you how much CPU time the process consumed. e.g. 1 second would be the process did computations for one second on one core.
I didn't find a separate percentage metric for that, but I think you can use this one. Just multiply the metric by 100.
e.g. `irate(process_cpu_seconds_total[1m]) * 100`.

Reply:
@tomashibm - you can talk with Tym about this to see what we are using over in our systems.